app/views/posts/_truncated.html.erb

```diff
@@ -0,0 +1,5 @@
+<div>
+  <h2><%= link_to post.title, post_path(post) %></h2>
+  <p><%= post.published_at %></p>
+  <p><%= truncate(post.article.content, length: 200, separator: ' ') { link_to t(".keep_reading"), post_path(post) }%></p>
```

Comment:
Wait, this is a built in Rails method? I've always done it manually 😲

Reply:
Yup!